context/brand-context/springernature/scss/30-mixins/links.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * Default link style
+ *
+ */
+
+@mixin base-link {
+	color: $context--link-color;
+	text-decoration: underline;
+	text-decoration-skip-ink: auto;
+
+	&.visited,
+	&:visited {
+		color: $context--link-visited-color;
+	}
+
+	&.hover,
+	&:hover {
+		color: $context--link-hover-color;
+		text-decoration: none;
+	}
+
+	&.active,
+	&:active {
+		color: $context--link-active-color;
+	}
+
+	&.focus,
+	&:focus {
+		outline: spacing(4) solid $context--focus-color;
+	}
+}
+
+/**
+ * Interface links
+ *
+ */
+
+@mixin interface-link {
```

Comment:
Do we still need this mixin now that we have `u-interface-link` in the default brand?
(obviously removing this will require changing all references to it)

Reply:
Good spot, will fix that